CraftTweaker2-API/src/main/java/crafttweaker/api/tooltip/IngredientTooltips.java

```diff
@@ -75,14 +81,17 @@ public static List<Pair<ITooltipFunction, ITooltipFunction>> getAdvancedShiftToo
     public static List<Pattern> getTooltipsToRemove(IItemStack item) {
         return REMOVED_TOOLTIPS.getEntries(item);
     }
-    
-    public static boolean shouldClearToolTip(IItemStack item) {
-        for(IIngredient cleared : CLEARED_TOOLTIPS) {
-            if(cleared.matches(item)) {
-                return true;
-            }
-        }
-        return false;
+
+    public static List<Integer> getTooltipLinesToRemove(IItemStack item) {
+        return REMOVED_TOOLTIPS_LINE.getEntries(item);
+    }
+
+    // Boolean wrapper as 3 status flag
+    // null -> don't clear tooltip, FALSE -> clear tooltip, TRUE -> clear tooltip but leave name
+    // a little dirty, but making an enum for the small thing is annoying
+    public static Boolean shouldClearToolTip(IItemStack item) {
```

Comment:
I think this would be a break from `boolean` -> `Boolean`, but I don't think anyone calls this method besides us, so I am okay with the break, but we may need to address it if any issues come up